Makefile

```diff
@@ -66,3 +66,19 @@ santacoder:
 .PHONY: optimized-santacoder
 optimized-santacoder:
 	${RUN_HF} --pretrained_model=olivierdehaene/optimized-santacoder --tokenizer=bigcode/santacoder --trust_remote_code ${EXP_ARGS}
+
+.PHONY: santacoder-cpu
```

Comment:
No need to have different targets for cpu and gpu, $EXTRA_ARGS already covers it. 
Why do we need `mayank31398/santacoder`? How is it different from the other targets above?
Please build the commands like the other ones above.

Reply:
`mayank/santacoder` has a fix for a bug. This bug is not encountered when running the vanilla generate method.
But for compiling the graph, onnx runs the model by passing dummy tensors and doesn't take the generate() path.
Its only a single line change in `modeling_gpt`

Reply:
I had created a PR for this: https://huggingface.co/bigcode/santacoder/discussions/23#640f55462ec22527ea436d5e